amundsen-kube-helm/templates/restore-backup/restore-neo4j-pod.yaml

```diff
@@ -2,6 +2,7 @@
     kind: Pod
     metadata:
       name: restore-neo4j-from-latest
+      annotations: fill_in_here_with_correct_annotation
```

Comment:
What will happen, if this is not set? I would think it would fail, since annotations expects a list of key value pairs, right?

Reply:
cc @higee

Reply:
Yeah, in case `annotations` is left empty, it fails with the following message.

```
│ Unable to locate credentials. You can configure credentials by running "aws configure".                                                                                                                  │
│ fatal error: Unable to locate credentials
```

For the full log from the `restore-neo4j-from-latest` pod, check the following

```
 fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz                                                                                                                              │
│ fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz                                                                                                                         │
│ (1/9) Installing nghttp2-libs (1.35.1-r2)                                                                                                                                                                │
│ (2/9) Installing libssh2 (1.9.0-r1)                                                                                                                                                                      │
│ (3/9) Installing libcurl (7.64.0-r3)                                                                                                                                                                     │
│ (4/9) Installing curl (7.64.0-r3)                                                                                                                                                                        │
│ (5/9) Installing expat (2.2.8-r0)                                                                                                                                                                        │
│ (6/9) Installing gdbm (1.13-r1)                                                                                                                                                                          │
│ (7/9) Installing python2 (2.7.18-r0)                                                                                                                                                                     │
│ (8/9) Installing py-setuptools (40.6.3-r0)                                                                                                                                                               │
│ (9/9) Installing py2-pip (18.1-r0)                                                                                                                                                                       │
│ Executing busybox-1.29.3-r10.trigger                                                                                                                                                                     │
│ OK: 145 MiB in 67 packages                                                                                                                                                                               │
│ Unable to locate credentials. You can configure credentials by running "aws configure".                                                                                                                  │
│ fatal error: Unable to locate credentials                                                                                                                                                                │
│ tar: short read                                                                                                                                                                                          │
│ ERROR (-v for expanded information):                                                                                                                                                                     │
│     /data (Is a directory)                                                                                                                                                                               │
│  -host      Domain name or IP of host to connect to (default: localhost)                                                                                                                                 │
│  -port      Port of host to connect to (default: 1337)                                                                                                                                                   │
│  -name      RMI name, i.e. rmi://<host>:<port>/<name> (default: shell)                                                                                                                                   │
│  -pid       Process ID to connect to                                                                                                                                                                     │
│  -c         Command line to execute. After executing it the shell exits                                                                                                                                  │
│  -file      File containing commands to execute, or '-' to read from stdin. After executing it the shell exits                                                                                           │
│  -readonly  Connect in readonly mode (only for connecting with -path)                                                                                                                                    │
│  -path      Points to a neo4j db path so that a local server can be started there                                                                                                                        │
│  -config    Points to a config file when starting a local server                                                                                                                                         │
│ Example arguments for remote:                                                                                                                                                                            │
│     -port 1337                                                                                                                                                                                           │
│     -host 192.168.1.234 -port 1337 -name shell                                                                                                                                                           │
│     -host localhost -readonly                                                                                                                                                                            │
│     ...or no arguments for default values                                                                                                                                                                │
│ Example arguments for local:                                                                                                                                                                             │
│     -path /path/to/db                                                                                                                                                                                    │
│     -path /path/to/db -config /path/to/neo4j.config                                                                                                                                                      │
│     -path /path/to/db -readonly                                                                                                                                                                          │
│ Welcome to the Neo4j Shell! Enter 'help' for a list of commands. Please note that neo4j-shell is deprecated and to be replaced by cypher-shell.                                                          │
│ NOTE: Remote Neo4j graph database service 'shell' at port 1337                                                                                                                                           │
│ neo4j-sh (?)$ CALL apoc.import.graphml('/data/', {useTypes: true, readLabels: tr ue});                                                                                                                   │
│ 10 ms                                                                                                                                                                                                    │
│ WARNING: Failed to invoke procedure `apoc.import.graphml`: Caused by: java.lang.RuntimeException: Error executing in separate thread: java.io.IOException: Cannot open file /data/ for reading.          │
│ stream closed
```